es6-modules/src/components/todo-form.js

```diff
@@ -0,0 +1,47 @@
+import createElement from '../lib/create-element';
```

Comment:
This is fine but it would have been better to use:
```JAVASCRIPT
import { createElement } from '../lib';
```
It is a bit shorter and is more flexible for maintainable and future updates